WordPress/Classes/Services/PostService.m

```diff
@@ -54,8 +54,8 @@ - (Post *)createPostForBlog:(Blog *)blog {
     post.postType = Post.typeDefaultIdentifier;
 
     BlogAuthor *author = [blog getAuthorWithId:blog.userID];
-    post.authorID = author.userID ?: blog.account.userID;
-    post.author = author.displayName ?: blog.account.displayName;
+    post.authorID = author.userID;
```

Comment:
Originally we fell back on the `blog.account.userID` if we had no hit with `getAuthorWithId`, but I can't determine why this is necessary. Tracking the author ID and display name is only useful to have for the UI to switch authors.

Therefore if we don't have authors to choose from, we won't be doing any switching.